lmfdb/inventory_app/jsonify_pg_structure.py

```diff
@@ -1,10 +1,19 @@
-import dbtools
-import id_object
+from id_object import get_description
```

Comment:
Shouldn't we be switching this to an absolute import?

Reply:
Agree